src/Microsoft.Sbom.Parsers.Spdx22SbomParser/Generator.cs

```diff
@@ -129,7 +129,7 @@ private void EnsureRequiredHashesPresent(Sbom.Contracts.Checksum[] fileHashes)
                 Created = identityUtils.GetGenerationTimestamp(internalMetadataProvider),
                 Creators = new List<string>
                 {
-                    $"Organization: {Constants.OrganizationValue}",
+                    $"Organization: {identityUtils.GetPackageSupplier(internalMetadataProvider)}",
```

Comment:
I know this isn't totally in the scope of this particular PR, but looking at `IdentityUtils` I'm not seeing the point of it not being a static class.  And if I'm correct, then all of it's methods can probably become extension methods.  That would result in you being able to have cleaner call here to `internalMetadataProvider.GetPackageSupplier()` 

If it's not worth doing in this PR, maybe refactoring separately would make sense.